pkg/microservice/aslan/core/common/repository/models/task/build.go

```diff
@@ -168,6 +168,20 @@ type JobCtx struct {
 	ClassicBuild    bool   `bson:"classic_build"                  json:"classic_build"`
 	PostScripts     string `bson:"post_scripts,omitempty"         json:"post_scripts"`
 	PMDeployScripts string `bson:"pm_deploy_scripts,omitempty"    json:"pm_deploy_scripts"`
+
+	// Upload To S3 related context
+	UploadEnabled     bool                              `bson:"upload_enabled"      json:"upload_enabled"`
+	UploadStorageInfo *ObjectStorageInfo                `bson:"upload_storage_info" json:"upload_storage_info"`
+	UploadInfo        []*models.ObjectStoragePathDetail `bson:"upload_info"         json:"upload_info"`
+}
+
+type ObjectStorageInfo struct {
```

Comment:
For new constructs that are used in more than one place, it is recommended to put these in `pkg/types/`.

Reply:
So now we just put all shared sturctures under one package?